lib/config/validation.ts

```diff
@@ -123,6 +129,19 @@ export async function validateConfig(
         message: `The "${key}" object can only be configured at the top level of a config but was found inside "${parentPath}"`,
       });
     }
+    if (key === 'enabledManagers' && val) {
+      const unsupportedManagers = getUnsupportedEnabledManagers(
+        val as string[]
+      );
+      if (is.nonEmptyArray(unsupportedManagers)) {
+        warnings.push({
```

Comment:
warning or error? The key question is whether we want to "stop" Renovate if it happens or not. Errors will stop and create an issue, while Warnings will not, and may be missed